Classes/Private/Manager/Video/OCTVideoEngine.h

```diff
@@ -0,0 +1,100 @@
+//
+//  OCTVideoEngine.h
+//  objcTox
+//
+//  Created by Chuong Vu on 6/19/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#import "OCTToxAV.h"
+
+@interface OCTVideoEngine : NSObject
+
+@property (weak, nonatomic) OCTToxAV *toxav;
+
+/**
+ * Current friend number that video engine should
+ * process video data to and from.
+ */
+@property (nonatomic, assign) OCTToxFriendNumber friendNumber;
+
+/**
+ * This must be called prior to using the video session.
+ * @param error Pointer to error object.
+ * @return YES if successful, otherwise NO.
+ */
+- (BOOL)setupWithError:(NSError **)error;
+
+/**
+ * Process incoming video frames.
+ * Set to YES to process video, otherwise NO
+ * to ignore the incoming frames in receiveVideoFrameWithWidth...
+ * Will be set to YES automatically when calling startSendingVideo
+ * Will be set to NO after stopSendingVideo
+ */
+@property (nonatomic, assign) BOOL processIncomingVideo;
+
+/**
+ * Start sending video data.
+ * This will turn on processIncomingVideo to YES
+ */
+- (void)startSendingVideo;
+
+/**
+ * Stop sending video data.
+ * This will turn off processIncomingVideo to NO
+ */
+- (void)stopSendingVideo;
```

Comment:
It would nice to know if video is being send. Maybe change `{start,stop}SendingVideo` to `sendVideo` property?